src/System.Runtime.InteropServices.RuntimeInformation/src/Architecture.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    internal enum Architecture
+    {
+        Undefined,
+        BIT32,
+        BIT64,
+        ARM32
+    }
+}
```

Comment:
This is an internal struct, will not be exposed to public surface area. So any naming is fine for me and can be changed.